src/rules/unused_macros.erl

```diff
@@ -78,3 +78,10 @@ ignored({Name, _Arity}, Name) ->
     true;
 ignored(_Pattern, _IgnoreSpec) ->
     false.
+
+is_member(_, []) ->
+    false;
+is_member(H, [{H, _} | _]) ->
+    true;
+is_member(H, [_ | T]) ->
+    is_member(H, T).
```

Comment:
Would this work if we make it like this…

```suggestion
is_member(MacroName, none, UsedMacros) ->
    lists:keymember(MacroName, 1, UsedMacros);
is_member(MacroName, MacroArity, UsedMacros) ->
    lists:member({MacroName, MacroArity}, UsedMacros).
```

…or like this…

```suggestion
is_member({MacroName, none}, UsedMacros) ->
    lists:keymember(MacroName, 1, UsedMacros);
is_member(Macro, UsedMacros) ->
    lists:member(Macro, UsedMacros).
```

…?

I mean… only macros with `none` arity can be used as function calls.

Reply:
I agree with this, checking `none` or the "old way"

Reply:
Done